coc/ext/fullwarapi/__init__.py

```diff
@@ -189,9 +189,7 @@ async def war_result_log(self, clan_tag: str) -> Optional[Generator[ClanWar]]:
                                    f"/war_result_log?clan_tag={correct_tag(clan_tag, '%23')}")
         try:
             responses = data["log"]
-
-            generator = (ClanWar(data=response["response"], client=self.coc_client) for response in responses)
-            return generator
+            return (ClanWar(data=response["response"], client=self.coc_client, clan_tag=coc.utils.correct_tag(clan_tag)) for response in responses)
```

Comment:
Idk much about generators, but this still looks like only one parameter to me. Where do the `None`s come from? And why do we need them in the first place?